queries/elixir.scm

```diff
@@ -0,0 +1,121 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(call
+  target: (_) @functionCallee
+) @functionCall @functionCallee.domain
+
+;; Map
+;; %{:a => "lorem", "b" => "ipsum", 3 => "dolor"}
+(map_content
+  (_)? @_.leading.start.endOf
+  .
+  (binary_operator) @collectionItem @_.leading.end.startOf @_.trailing.start.endOf
+  .
+  (_)? @_.trailing.end.startOf
+  (#insertion-delimiter! @collectionItem ", ")
```

Comment:
this won't behave nicely in a vertical list, eg if you have newlines between elements. I'm hesitant to introduce poor clone / pour behaviour for a new language. Maybe you can drop into a meet-up and we can discuss how to do this one right? It hasn't yet been done for any next-gen language

Reply:
Yeah sounds good, I'll be at the next Thursday one

Reply:
I would like to discuss drink and pour as well. I can’t make Thursdays, but hopefully you all can tolerate having multiple conversations about it.